dev_notes/dev-quick-start.md

```diff
@@ -0,0 +1,55 @@
+
+#### Prerequisites
+
+* install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+* install python dependencies ```pip install -r requirements.txt```
```

Comment:
nit: should we add an intermediate step about creating a `virtualenv` for python?

Reply:
yep not a bad idea, will add